overlord/ifacestate/handlers.go

```diff
@@ -458,13 +444,9 @@ func (m *InterfaceManager) doDisconnect(task *state.Task, _ *tomb.Tomb) error {
 		return err
 	}
 
-	affectedConns, err := m.repo.ResolveDisconnect(plugRef.Snap, plugRef.Name, slotRef.Snap, slotRef.Name)
```

Comment:
Ok, as discussed in the standup this is a no-issue after all, we will let disconnect fail in such rare cases. No need for patching.